### PR TITLE
feat: decouples telemetry from agent

### PR DESCRIPTION
**Pull request by prajjwaldimri**

### CODE REVIEW

File: tools/launchd/com.warpbuild.warpbuild-telemetryd.plist [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>Label</key>
+    <string>warpbuild-agentd-launcher</string>
```
Comment (PrashantRaj18198) [inline tools/launchd/com.warpbuild.warpbuild-telemetryd.plist:6]: label change required.

File: tools/launchd/com.warpbuild.warpbuild-telemetryd.plist [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <integer>15</integer>
+    <key>KeepAlive</key>
+    <dict>
+      <key>SuccessfulExit</key>
+      <false/>
+    </dict>
+    <key>StandardErrorPath</key>
+    <string>$BASE_DIR/log/warpbuild-agentd-launcher.error.log</string>
+    <key>StandardOutPath</key>
+    <string>$BASE_DIR/log/warpbuild-agentd-launcher.out.log</string>
```
Comment (PrashantRaj18198) [inline tools/launchd/com.warpbuild.warpbuild-telemetryd.plist:32]: error path changes required.

File: tools/launchd/com.warpbuild.warpbuild-telemetryd.plist (hunk 1 above)
Comment (prajjwaldimri) [reply tools/launchd/com.warpbuild.warpbuild-telemetryd.plist:6]: Done

File: tools/launchd/com.warpbuild.warpbuild-telemetryd.plist (hunk 2 above)
Comment (prajjwaldimri) [reply tools/launchd/com.warpbuild.warpbuild-telemetryd.plist:32]: Done

**Review by PrashantRaj18198 — APPROVED** [review]
lgtm 🪭

**Pull request merged**